dashboard-dataviz/zimbabwe_dashboard/01_preparing_data_for_dashboard/02_clean_telecom_data/clean_subscribers_data.R

```diff
@@ -2,6 +2,9 @@
 
 EXPORT <- T
 
+
+unit <- "district"
```

Comment:
```suggestion

```

Reply:
For some reason enabling the password wasn't working when things were split up into multiple files, but putting everything back into one file fixed it. Not sure why. Might have something to do with two separate "UIs" being defined (one for the password landing page, and one for the main page), but not exactly sure why things don't work when split up.